src/components/Quiz.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import Answers from './Answers';
+import { publish } from './Event';
+import QuestionAnswer from './QuestionAnswer';
+import Score from './Score';
+import TimerContainer from './TimerContainer';
+
+function Quiz() {
+  const [questionNumber, setQuestionNumber] = useState(1);
+  const [questions, setQuestions] = useState([]);
+  const nextQuestion = (question) => {
+    if (questionNumber < 21) {
+      setQuestions([...questions, question]);
+      publish('ResetInterval', 20);
+      setQuestionNumber(questionNumber + 1);
+    }
+  };
+  const shouldShowQuiz = questionNumber < 21;
```

Comment:
We should maintain a state variable for this

Reply:
And only change once we have submitted the final question

Reply:
Maintaining the current question state from which we can get the question number.